mobify/astro-scaffold
# HYB-723: IOS layout can be configured to use tabs or drawer

iOS layout can be configured to use tabs or drawer

JIRA: https://mobify.atlassian.net/browse/HYB-723
Linked PRs: https://github.com/mobify/generator-astro/pull/21

## Changes
- Add a boolean to baseConfig to allow configuration of ios layout
- Fix the drawer so that it works in ios (I know we are removing this soon but oh well. Wasn't long change. Might as well have it functional for now)
- Added a really ugly drawer icon to ios. We need a better one!

## How to test-drive this PR
- Test by setting the `iosUsingTabLayout` in `baseConfig` to true and false. 
- Ensure that both layouts work in ios
- Ensure that no matter what the flag is set to drawer layout always appears on android
- Test that the drawer functions correctly on both platforms

## TODOS:
